buildSrc/src/main/resources/META-INF/schemas/guide-metadata.schema.json

```diff
@@ -0,0 +1,167 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "uri": "https://guides.micronaut.io/schemas",
+    "type": "object",
+    "properties": {
+        "minimumJavaVersion": {
+            "type": "integer"
+        },
+        "prerequesites": {
+            "type": "array",
+            "items": {
+                "type": "string"
+            }
+        },
+        "clouds": {
```

Comment:
"cloud" we have `cloud` singular. https://github.com/micronaut-projects/micronaut-guides/pull/1491/files#diff-a9109f731ed2d376b8c7828e2dc99586ef4aacb1d5057e7cb3227a30d4730b47